openpype/tools/launcher/actions.py

```diff
@@ -74,7 +81,7 @@ class ApplicationAction(api.Action):
     @property
     def log(self):
         if self._log is None:
-            self._log = Logger().get_logger(self.__class__.__name__)
+            self._log = Logger.get_logger(self.__class__.__name__)
```

Comment:
Good catch - both should work but it's definitely easier to read and more explicit that we're using (in this case) a classmethod. A quick search over the code base does show that `Logger().get_logger(`  is used in many more places. Would it be worth it to remap all of them for consistency (potentially separate PR of course)

Reply:
Yes, but I would wait after refactor. We will move code from place to place during the refactor.